src/test/java/io/debezium/connector/yugabytedb/YugabyteDBUserDefinedTypesTest.java

```diff
@@ -0,0 +1,113 @@
+package io.debezium.connector.yugabytedb;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+import org.apache.kafka.connect.source.SourceRecord;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.YugabyteYSQLContainer;
+
+import io.debezium.config.Configuration;
+import io.debezium.connector.yugabytedb.common.YugabyteDBTestBase;
+import io.debezium.util.Strings;
+
+/**
+ * Basic unit tests to check the behaviour of User Defined types with CDC in YugabyteDB.
+ * 
+ * @author Vaibhav Kushwaha (vkushwaha@yugabyte.com)
+ */
+@Ignore
+public class YugabyteDBUserDefinedTypesTest extends YugabyteDBTestBase {
+  private final static Logger LOGGER = LoggerFactory.getLogger(YugabyteDBUserDefinedTypesTest.class);
+  private static YugabyteYSQLContainer ybContainer;
+
+  // Fully qualified table name
+  private final String FQ_TABLE_NAME = "public.udt_table";
+
+  private final String INSERT_FORMAT = "INSERT INTO udt_table VALUES (%s, ('Vaibhav', 'Kushwaha'));";
+
+  @BeforeClass
+  public static void beforeClass() throws Exception {
+    // Make sure this test is using the YugabyteDB version which has the changes to support
+    // composite types with Change data capture
+    // ybContainer = TestHelper.getYbContainer();
+    // ybContainer.start();
+
+    // TestHelper.setContainerHostPort(ybContainer.getHost(), ybContainer.getMappedPort(5433));
```

Comment:
Commented this to use a local build with the server side changes for UDT support. Will uncomment and change the docker image version so as to use the latest image with all the necessary changes.